Xamarin.Forms.Core/CarouselView/CarouselView.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Collections;
+
+namespace Xamarin.Forms
+{
+	/// <summary>
+	/// CarouselView Interface
+	/// </summary>
+	public class CarouselView : View //Layout<View>
```

Comment:
Not requiring derivation of ListView certainly allows for a simpler implementation.